### ISSUE TITLE
Windows 7 jump list

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mintty/mintty#290]
```
Would be great to have a couple of options in the Windows 7 'jump-list' (The 
menu that's displayed when you right-click mintty in the taskbar).

A couple if options that'd be handy would be things like:
- Open new window
- Run script from file (Probably another enhancement altogether)
- Open in new shell environment (bsh, ksh, etc.)
- Options 

With Regards,

James
```

Original issue reported on code.google.com by `VWJ...@gmail.com` on 7 Sep 2011 at 4:41

**Comment by mintty** [comment]
I've tried this patch on the occasion of checking another taskbar-related issue (#471) but it fails at pCustomDestinationList->lpVtbl->SetAppID.

**Comment by mintty** [comment]
Modified patch to adapt to wchar API and retried.
SetAppID fails with ERROR_NO_TOKEN.
(If anyone wants to give it a try, I'll provide the adapted patch here.)

**Comment by mintty** [comment]
The new Virtual Tabs feature (2.8.0, #8) provides a session launcher.
Its configuration could be reused to populate a taskbar jump list.
Implementation problems still persist.

**Comment by mintty** [comment]
Uploaded jump list support.
This was verified on a Windows 10 and a Windows 7 system but fails on another Windows 7 system for unknown reason.

**Comment by mintty** [comment]
Released 2.8.4.
